src/generators/static/packageFileGenerator.ts

```diff
@@ -27,7 +27,8 @@ export function generatePackageJson(
     dependencies: {
       ...(hasLRO && { "@azure/core-lro": "^1.0.1" }),
       ...(hasAsyncIterators && { "@azure/core-paging": "^1.1.1" }),
-      "@azure/core-http": "^1.1.4",
+      "@azure/core-client": "1.0.0-beta.1",
+      "@azure/core-https": "^1.0.0-beta.2",
```

Comment:
heads up this is getting a rename: https://github.com/Azure/azure-sdk-for-js/pull/14042

Reply:
I will update this PR once the rename PR is merged

Reply:
btw the rename has been merged.

Reply:
Yeah. It was merged yesterday. I am working on resolving the test issues. Along with that I will do the rename also